packages/backend/src/apps/lettersg/auth/verify-api-key.ts

```diff
@@ -0,0 +1,14 @@
+import { IGlobalVariable } from '@plumber/types'
+
+export async function verifyApiKey($: IGlobalVariable): Promise<void> {
+  try {
+    await $.http.get('/v1/templates?limit=1')
+  } catch (err) {
+    if (err.response.status === 401) {
+      throw new Error(
```

Comment:
comment: maybe we should enhance StepError to make step number optional <.<

Reply:
i think for now, `StepError` should be just for testing step but can look into this in the future!